traitsui/tests/editors/test_liststr_editor_selection.py

```diff
@@ -74,21 +92,539 @@ class ListStrEditorWithSelectedIndex(HasTraits):
 )
 
 
-def get_selected(control):
+def get_selected_indices(editor):
     """ Returns a list of the indices of all currently selected list items.
     """
-    import wx
+    if is_current_backend_wx():
+        import wx
+
+        item = -1
+        selected = []
+        while True:
+            item = editor.control.GetNextItem(
+                item, wx.LIST_NEXT_ALL, wx.LIST_STATE_SELECTED
+            )
+            if item == -1:
+                break
+            selected.append(item)
+        return selected
+
+    elif is_current_backend_qt4():
+        indices = editor.list_view.selectionModel().selectedRows()
+        return [i.row() for i in indices]
+
+    else:
+        raise unittest.SkipTest("Test not implemented for this toolkit")
+
+
+def set_selected_single(editor, index):
```

Comment:
Oh yes, completely forgot about docstrings! 🤦

I checked and I can remove `clear_selection` from wx branch because it is somehow integrated in the Select function when `muli_select=False`. Qt code more explicitly integrates clearing the selection.